### PR TITLE
Update Apache Cassandra

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
apachecassandra

**Issue:** contributes toward #9072 

**Similarweb rank:** --. See https://www.similarweb.com/website/cassandra.apache.org

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://svn.apache.org/repos/asf/cassandra/logo/cassandra.svg (Updated)
Color: `#1287b1` (Unchanged)

### Note
Downloading from here just gives me a white box https://simpleicons.org/icons/apachecassandra.svg
Clicking on Download PDF also generates an error for me.
I don't know if it is just my system or some error with the icon in simple-icons repository. I wanted to do a comparison to show the corrections I made. 
comparison

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -875,7 +875,7 @@
         {
             "title": "Apache Cassandra",
             "hex": "1287B1",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/5/5e/Cassandra_logo.svg"
+            "source": "https://svn.apache.org/repos/asf/cassandra/logo/cassandra.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:878]:
```suggestion
            "source": "https://www.apache.org/logos"
```

Make sure your submitted icon is based on this updated source (the v4 version!), this is where we should pull our Apache project icons from.

Comment (PratyushJayachandran) [reply _data/simple-icons.json:878]: I have redone the SVG from this source although it seems to be the same.

**Review by adamrusted — APPROVED** [review]
This looks spot on to me. For reference @simple-icons/icon-review - there is a minor inconsistency [above the eyelashes](https://github.com/simple-icons/simple-icons/assets/1830867/47835a58-37f4-4658-bf74-b3df7a873c9f) in the original, so the light blue of the eye bleeds through. Seems @PratyushJayachandran has managed to tidy up their icon for them in the process of submitting this PR! 🤣🎉

**Comment by PratyushJayachandran** [comment]
> This looks spot on to me. For reference @simple-icons/icon-review - there is a minor inconsistency [above the eyelashes](https://github.com/simple-icons/simple-icons/assets/1830867/47835a58-37f4-4658-bf74-b3df7a873c9f) in the original, so the light blue of the eye bleeds through. Seems @PratyushJayachandran has managed to tidy up their icon for them in the process of submitting this PR! 🤣🎉

Hah I deleted those because it looked so odd in monochrome 🤣

**Pull request merged**